src/closeLoginPopup.js

```diff
@@ -0,0 +1,30 @@
+const closeLoginPopupOpenTargets = document.querySelectorAll('[data-login-popup-close-target]');
+let loginOpen = 0;
+
+const attachBackgroundListener = (event) => {
```

Comment:
Define handleLoginPopupBackgroundClick.js for this and attach it to DOM when popup is opened and remove it when the popup is closed